single-kernel/aes_encrypt.cpp

```diff
@@ -0,0 +1,477 @@
+#include <sycl.hpp>
```

Comment:
Correct namespace would `sycl/sycl.hpp`.

Reply:
Done [Changed include sycl to sycl/sycl.hpp](https://github.com/unisa-hpc/sycl-bench/pull/56/commits/02eff75e1cad8d4c95867014cd7915734fe1d07c)